doc/reference/reference_lua/box_ctl/demote.rst

```diff
@@ -0,0 +1,28 @@
+.. _box_ctl-demote:
+
+box.ctl.demote()
+=================
+
+..  function:: demote()
+
+    Revoke the leader role of the instance.
```

Comment:
_From_ the instance sounds more natural to me. I've found an [example](https://docs.aws.amazon.com/redshift/latest/dg/r_REVOKE.html): "The role that you revoke from a specified user" 

```suggestion
    Revoke the leader role from the instance.
```

Reply:
Sounds good! Thank you for the example.